cluster/stone-prd-rh01/managed/rh-managed-cnv-fbc-tenant/release_plan_admission.yaml

```
@@ -4,47 +4,56 @@ metadata:
   name: fbc-release
   namespace: rh-managed-cnv-fbc-tenant
 spec:
-  application: cnv-fbc
+  applications:
+    - cnv-fbc
+  data:
+    fbc:
+      binaryImage: "registry.redhat.io/openshift4/ose-operator-registry:v4.12"
+      buildTimeoutSeconds: 480
+      fromIndex: "quay.io/scoheb/fbc-index-testing:latest"
+      requestUpdateTimeout: 420
+      targetIndex: "quay.io/scoheb/fbc-target-index-testing:latest"
+  pipelineRef:
+    resolver: git
+    params:
+      - name: url
+        value: "https://github.com/redhat-appstudio/release-service-catalog.git"
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: "pipelines/fbc-release"
   origin: cnv-fbc-tenant
-  releaseStrategy: fbc-strategy
+  policy: rh-fbc-policy
+  serviceAccount: release-service-account
 ---
 apiVersion: appstudio.redhat.com/v1alpha1
 kind: ReleasePlanAdmission
 metadata:
   name: fbc-staged-index-release-v4-14
```

```suggestion
  name: fbc-staged-index-release
```
since we now have all application under this RPA, we should remove the version tag in the name

---

Changed.